python/lsst/pipe/base/argumentParser.py

```diff
@@ -1252,6 +1243,35 @@ def __call__(self, parser, namespace, values, option_string):
         ident.idList += idDictList
 
 
+class LongLogAction(argparse.Action):
+    """argparse action to make logs verbose.
+
+    An action so that it can take effect before log level options.
```

Comment:
Does this actually mean "take effect before log level options that appear after --longlog"? I think this still does not handle `--loglevel=debug --longlog` ordering as expected. Is this what you meant by "good-enough fix"?

Reply:
Yes, it does not handle `--loglevel` options that are placed before.  It's not great, but it's simpler than arranging to save all the loglevel arguments and play them after configuring longlog (which is what I think you do in Gen3).